NetKVM/Common/ParaNdis-TX.h

```diff
@@ -60,10 +62,15 @@ class CNB : public CNdisAllocatableViaHelper<CNB>
     USHORT QueryL4HeaderOffset(PVOID PacketData, ULONG IpHeaderOffset) const;
     void DoIPHdrCSO(PVOID EthHeaders, ULONG HeadersLength) const;
     void SetupCSO(virtio_net_hdr *VirtioHeader, ULONG L4HeaderOffset) const;
-    bool FillDescriptorSGList(CTXDescriptor &Descriptor, ULONG DataOffset) const;
+    bool FillDescriptorSGList(CTXDescriptor &Descriptor, ULONG DataOffset);
     bool MapDataToVirtioSGL(CTXDescriptor &Descriptor, ULONG Offset) const;
     void PopulateIPLength(IPHeader *IpHeader, USHORT IpLength) const;
 
```

Comment:
style: do we want a new line here?

Reply:
I'm not sure if I should add newline here, what do you think?
If we add a new line, is it necessary to move the position of this function after PopulateIPLength?